src/GraphQL.Relay/Types/NodeGraphType.cs

```diff
@@ -15,7 +15,7 @@ public class GlobalId
 
     public interface IRelayNode<out T>
     {
-        T GetById(string id);
+        T GetById(string id, IResolveFieldContext<object> context);
```

Comment:
Do you think the context should come first?  Or is it better with the id first?

Also if we are not providing a strongly typed `IResolveFieldContext<T>` then we should probably just use `IResolveFieldContext` rather than `IResolveFieldContext<object>`

Reply:
Good point on the interface. I was struggling with the same thing as far as ordering.
Honestly, it should probably be first? Since that seems to be the convention for the rest of the graphql-dotnet libraries.

Reply:
Another slightly weird thing that I was struggling with is if someone accessed the context and grabbed the "Id" argument. Since that would expose some of the internal work that graphql-relay is doing. Not sure if that's a big deal.

Reply:
I think having the context first makes sense. But either way is probably fine. 

I wouldn’t worry about someone reading the id from the context. If someone goes to that trouble, then they probably have a good reason to. And any implementation here would be server side code - there are no “secrets” to hide from an implementation.

Reply:
So are we sticking with id first or putting the context first?

Reply:
Been busy. If you want I can flip it and put the ID first. I think it makes more sense that way

Reply:
You mean the context first?  I think the id is first now.  I'm okay either way.